### ISSUE TITLE
Motion Debluring Train

### ISSUE THREAD

**Issue by niehen6174** [swz30/Restormer#54]
Hi.Thank you so much for your open source work. When I trained motion_deblur, I found that the effect in the paper could not be achieved.
1. I followed the dependency tutorial mentioned in the repository ,downloaded the gopro dataset, and used the provided crop method to prepare the training set and validation set.
2. And use the Deblurring_Restormer.yml configuration file for training.  In the configuration file I modified to use single GPU training. 
3. In another experiment, I modified the training strategy to fix the crop size to 128.
But the results of both experiments were less than 31db, which was much lower than the results in the paper.
I wonder if details are missing and why the results are so different.

**Comment by morgen-star** [comment]
> Hi.Thank you so much for your open source work. When I trained motion_deblur, I found that the effect in the paper could not be achieved.
> 
> 1. I followed the dependency tutorial mentioned in the repository ,downloaded the gopro dataset, and used the provided crop method to prepare the training set and validation set.
> 2. And use the Deblurring_Restormer.yml configuration file for training.  In the configuration file I modified to use single GPU training.
> 3. In another experiment, I modified the training strategy to fix the crop size to 128.
>    But the results of both experiments were less than 31db, which was much lower than the results in the paper.
>    I wonder if details are missing and why the results are so different.

Hello,  whether you reduce the batch size or not  when you use a single GPU to train?  In my opinion, reducing batch time may decrease the performance.   (I am not the author)

**Comment by Mytttttttt** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by Andrew0613** [comment]
Did you find the solution? I met the same problem on Gaussian Denoising.
